timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -290,6 +303,20 @@ def _create_target_scaler(self) -> Optional[LocalTargetScaler]:
         else:
             return None
 
+    def _create_covariate_scaler(self) -> Optional[CovariateScaler]:
+        """Create a CovariateScaler object based on the value of the `covariate_scaler` hyperparameter."""
+        covariate_scaler_type = self._get_model_params().get("covariate_scaler")
```

Comment:
just to check: is `"global"` set as the default somewhere, i.e., are we setting it as the default for all `AbstractTimeSeriesModel`s or not? (I assume not)

Reply:
We only set it as default for GluonTS models (https://github.com/autogluon/autogluon/pull/4634/files#diff-28a0cc813c0c8a1cd39a4e8c358ee689718114207a2655af00cb4ae7675d7665R293) so the behavior is equivalent to before the refactor